.gitignore

```diff
@@ -5,7 +5,7 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
+#*.jar
```

Comment:
this should be added back. We should have @lewismc or @sujen1412 publish the protocol-interactiveselenium on the Central repo